301-quarkus-vertx-kafka/src/test/java/io/quarkus/qe/vertx/kafka/SchemaRegistryContainer.java

```diff
@@ -0,0 +1,43 @@
+package io.quarkus.qe.vertx.kafka;
+
+
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.KafkaContainer;
+import org.testcontainers.containers.Network;
+
+public class SchemaRegistryContainer extends GenericContainer<SchemaRegistryContainer> {
+
+    public SchemaRegistryContainer(String version) {
+        super("confluentinc/cp-schema-registry:" + version);
+        withExposedPorts(8081);
+    }
+
+    public SchemaRegistryContainer withKafka(KafkaContainer kafka) {
+        return withKafka(kafka.getNetwork(), kafka.getNetworkAliases().get(0) + ":9092");
+    }
+
+    public SchemaRegistryContainer withKafka(Network network, String bootstrapServers) {
+        withNetwork(network);
```

Comment:
I would join these two methods in one, also the withNetwork is not necessary (it's done before calling this method, and also the KafkaContainer might be running in another network -> It's not in the test, but it could be)

Reply:
I like current way, combining `withStrimziKafka` and `withConfluentKafka` into one method would make the readability worse.